src/NUnitConsole/nunit3-console.tests/ResultReporterTests.cs

```diff
@@ -100,9 +100,10 @@ public void SummaryReportTest()
         {
             var expected = new string[] {
                 "Test Run Summary",
-                "    Overall result: Failed",
-                "   Tests run: 35, Passed: 30, Errors: 1, Failures: 3, Inconclusive: 1",
-                "     Not run: 10, Invalid: 3, Ignored: 4, Explicit: 3, Skipped: 0",
+                "  Overall result: Failed",
+                "  Tests run: 35, Passed: 30, Failed: 7, Inconclusive: 1, Skipped: 7",
```

Comment:
The number 35 no longer makes sense here. Previously it was the sum of the other items on the line, that is passed + errors + failures + inconclusive. I guess it should now be the count of  test cases, which is 45, the total of the earlier tests run and not run. @rprouse What do you think?